src/main/java/science/icebreaker/dao/entity/ResetPasswordToken.java

```diff
@@ -0,0 +1,57 @@
+package science.icebreaker.dao.entity;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+
+@Entity
+public class ResetPasswordToken {
+    @Id
+    private String token;
```

Comment:
Using the account as id would enforce only one valid token at the same time.

Reply:
I could go with a unique index as well